src/attributes/classes.js

```diff
@@ -7,16 +7,20 @@ define([
 
 var rclass = /[\t\r\n\f]/g;
 
+function getClass( elem ) {
+	return elem.getAttribute && elem.getAttribute( "class" ) || "";
```

Comment:
We need the check because we have tests supporting some silent fails. The proposed alternative differs in logic. This way still returns an empty string in the case where getAttribute does not exist.